packages/shared/lib/core/layer-2/interfaces/layer2-asset-allowance.interface.ts

```diff
@@ -1,7 +1,7 @@
 import { NativeTokenAmount } from '@core/token'
 
 export interface ILayer2AssetAllowance {
-    baseTokenAmount: string
+    baseTokens: string
```

Comment:
Seems this name should be singular rather than plural?

Reply:
I agree with you, but this is not the object that the WASP node expects, so we have to request some changes.

Reply:
It's even uglier for `nativeTokens`, because the ID property expects a string tuple. I'll open an issue in WASP, since they might change this.